### PR TITLE
Migrate from `go_module()` to `go_get()`

### PR DESCRIPTION
**Pull request by Tatskaari**
Use the new experimental go_get() rules that compile third party code as a subrepo. We will need a better way to update these targets as the new labels are quite unweildly.

### CODE REVIEW

File: src/parse/asp/BUILD [hunk 1]
```diff
@@ -2,37 +2,58 @@ subinclude("//build_defs:benchmark")
...
+        "config_test.go",
+        "file_position_test.go",
+        "interpreter_test.go",
+        "label_context_test.go",
+        "lexer_test.go",
+        "logging_test.go",
+        "parser_test.go",
+        "targets_test.go",
+        "util_test.go",
+    ],
```
Comment (samwestmoreland) [inline src/parse/asp/BUILD:48]: How come the globs are expanded? Is that expected?

**Review by samwestmoreland — APPROVED** [review]
Exciting!

**Review by peterebden — APPROVED** [review]
Seems good! I think (hope) this will be a lot less work to maintain `//third_party/go` although arguably it's a bit more to add deps for each library (but maybe they are more systematic so still okay)

**Comment by Tatskaari** [comment]
> Seems good! I think (hope) this will be a lot less work to maintain `//third_party/go` although arguably it's a bit more to add deps for each library (but maybe they are more systematic so still okay)

There's an install` arg on the `go_get()` rules which makes them behave similarly to the `go_module()` rules, i.e. you can depend on `//third_party/go:testify`. I have already done the migration here though. We could start using them going forward if this is too much of a pain, but I'd like to use the Please repo as a test bed for writing a tool to automatically generate the library rules for us.

**Pull request merged**